### PR TITLE
Feature: action versions

### PR DESCRIPTION
**Pull request by rgerstenberger**
This adds the action `versions` which outputs any modules that are included in the bundle more than once and where that version skew is coming from.

Remaining tasks:

* [x] Text output
* [x] TSV output 
* [x] Filter to duplicates with `--duplicates` flag

/cc @ryan-roemer

### CODE REVIEW

**Comment by ryan-roemer** [comment]
@rgerstenberger -- I've pulled down the code and am running:

```
$ bin/inspectpack.js \
  --action=versions \
  --bundle="/PATH/TO/BUNDLE" \
  --format=json
```

But I get this output:

```
[]
```

What command should I do to get actual output?

**Comment by ryan-roemer** [comment]
@rgerstenberger - did you get a chance to look at this yet? Would love an update / look at what I'm doing wrong here!

**Comment by rgerstenberger** [comment]
Ah sorry @ryan-roemer! I should put some error handling in this as well. I had to debug this manually to find the error which is that `"/PATH/TO/PROJECT"` is not the webpack root, it's actually `"/PATH/TO/PROJECT/src"`

`root` is the folder of the webpack entry point. How should I document and/or update the code/naming to make that clearer?

**Comment by ryan-roemer** [comment]
@rgerstenberger -- Ah, yeah `src` was totally it!

So, running `--action=versions` and `--action=duplicates`, I'm confused by the results. For just one item, `object-assign` I have these results:

**Versions**:

```js
  {
    "name": "object-assign",
    "skewedDeps": [],
    "versions": {
      "3.0.0": [
        "root -> @foo/baz@4.0.0 -> eventuate@4.0.0"
      ],
      "4.0.1": [
        "root -> @foo/baz@4.0.0"
      ],
      "2.1.1": [
        "root -> @foo/bar-product-card@4.0.0 -> @foo/bar-product-buttons@4.6.0 -> @foo/bar-base@2.1.1 -> @foo/bar-interactive@3.1.1 -> @foo/bar-layout@1.0.6"
      ]
    }
  },
```

**Duplicates**:

```md
* object-assign/index.js
    * Num Duplicates:          3
    * Current Bytes (min):     --
    * Extra Bytes (min):       --
    * Current Bytes (min+gz):  --
    * Extra Bytes (min+gz):    --

    * 542 (5): ../~/@foo/bar-base/~/object-assign/index.js
        * ../~/@foo/bar-interactive/~/object-assign/index.js
        * ../~/@foo/baz/~/object-assign/index.js
        * ../~/nuka-carousel/~/object-assign/index.js
        * ../~/@foo/bar-product-higher-order-components/~/@foo/bar-product-card/~/object-assign/index.js
        * ../~/react-shuffle/~/object-assign/index.js

    * 666 (7): ../~/radon-select/~/object-assign/index.js
        * ../~/@foo/bar-product-buttons/~/object-assign/index.js
        * ../~/@foo/bar-product-typography/~/object-assign/index.js
        * ../~/@foo/bar-product-offers/~/@foo/bar-base/~/object-assign/index.js
        * ../~/@foo/bar-product-offers/~/@foo/bar-containers/~/object-assign/index.js
        * ../~/@foo/bar-product-offers/~/@foo/bar-product-buttons/~/object-assign/index.js
        * ../~/@foo/bar-product-higher-order-components/~/object-assign/index.js
        * ../~/@foo/bar-carousel/~/object-assign/index.js

    * 1742 (0): ../~/object-assign/index.js
```

Can you walk me through both the above results? Because it seems like there are a lot more libraries containing `object-asign` (`radon-select`, `nuka-carousel`, etc.) that aren't accounted for in `versions`...

**Comment by rgerstenberger** [comment]
@ryan-roemer good catch! That was a definite bug in my code and I've pushed a fix.

File: lib/actions/versions.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    }, {}),
+
+    // Only keep modules with more than one version
+    getSkewedPackageMap
+  )(codes);
+
+  // If we have a rootPackageJSON, we can find dependency chains for all skewed deps.
+  // If not, throw.
+  if (!rootPackageJSON) {
+    throw new Error("Unable to find project root package.json");
```
Comment (ryan-roemer) [inline lib/actions/versions.js]:
```js
callback(new Error("Unable to find project root package.json"));
return;
```

Comment (rgerstenberger) [inline lib/actions/versions.js]: can do

**Comment by ryan-roemer** [comment]
@rgerstenberger -- Back to you for final review!!!

**Comment by rgerstenberger** [comment]
LGTM! Thanks for wrapping this up!

**Comment by ryan-roemer** [comment]
Released in `inspectpack@0.6.0`

**Pull request merged**